app/(tabs)/outfits.tsx

```diff
@@ -1,192 +1,44 @@
-import { Button, ScrollView, StyleSheet } from 'react-native';
-import { Text, View } from '@/components/Themed';
-
-import React, {useState} from 'react';
-import {Alert, Modal, Pressable, FlatList, TouchableOpacity } from 'react-native';
-
-import EditOutfits from '@/app/edit'
-
-/* To Do: 
-- add photos of "outfits" inside the button on the left side (--> *photo* <-- Outfits)
-- add functionality in Edit Outfit Page: 
-    -> eliminate outfits - delete feature
-    -> move outfit button - arrange outfits feature
-- continue implementing AddOutfit / Make your Outfit feature:
-  - add a filters search bar: includes three dropdown: Color, Type, Sleeve Size, etc
-  - has photos of their closet / database
-
-  --see https://www.figma.com/file/ojPtY7fqcm1botgu9QryZf/PhotoFit-UI?type=design&node-id=0-1&mode=design&t=xGQxI8btmzRFtTNz-0
-
-
-*/
-
-
-// Add Outfit Button
-function addOutfitButton(){
-
-  return(
-    <View style={{paddingBottom:5}}>
-    {/* Separator */}
-      <View style={styles.separator} lightColor="#ffff" darkColor="rgba(255,255,255,0.1)" />
-
-    {/* Button */}
-      <View style={styles.OutfitButton}> 
-        <Pressable  
-          // onPress={() => setModalVisible(true)}
-          >
-            <Text style={styles.textStyle}>Outfit</Text>
-        </Pressable>
-      </View>
-
-
-    </View>
-
-  );
-}
-
-
+import { ScrollView, StyleSheet, View, TouchableOpacity, Text } from 'react-native';
+import React from 'react';
+import OutfitComponent from "@/components/outfitWidget"; // Import the OutfitComponent
```

Comment:
Unused import, please fix!

Reply:
The outfit component must be used for this part. I turned it off because I need an outfit to display without squigly lines.

Reply:
Fixed